pymbar/mbar.py

```diff
@@ -1292,10 +1345,14 @@ def computePMF(self, u_n, bin_n, nbins, uncertainties='from-lowest', pmf_referen
 
         Returns
         -------
-        f_i : np.ndarray, float, shape=(K)
-            f_i[i] is the dimensionless free energy of state i, relative to the state of lowest free energy
-        df_i : np.ndarray, float, shape=(K)
-            df_i[i] is the uncertainty in the difference of f_i with respect to the state of lowest free energy
+        result_vals : dictionary
+
+        Possible keys in the result_vals dictionary:
+
+        'f_i' : np.ndarray, float, shape=(K)
```

Comment:
Is there a good reason this is `f_i` and `df_i` instead of `Delta_f` and `dDelta_f` like in `getFreeEnergyDifferences`?

Reply:
For a PMF, you are required to set an overall reference.  So it's IS a dF from the reference, but not from a specific state.